frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/form/validators.ts

```diff
@@ -30,12 +30,11 @@ export const dns1035Validator: IValidator = {
   help: $localize`Name must consist of lowercase alphanumeric characters or '-',
     start with an alphabetic character, and end with an alphanumeric character.`,
 };
-
+// aaw verify if i18n works
```

Comment:
comment isn't clear, it seems all we did here is add the i18n. You want us to test if it works?

Reply:
Forgot to remove... REmoving